### DISCUSSION TITLE
회차 상세페이지 내 기록 부분 문구 추가 논의

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#229] [Features]
회차 상세페이지에서 기록이 개인만을 위한 기록인지, 아니면 해당 회차의 모두를 위한 기록인지 사용자가 인지하기 어렵다는 피드백을 받았습니다. (실제로 저희도 논의했던 기억이 있네요)
저도 비슷하게 생각하는데, 이 부분은 _스터디원 모두가 볼 수 있음을 알리는 문구가 추가_ 되어야 한다고 생각해요.
위 같은 피드백을 받고나서 토스를 사용하다가 모임통장 내 거래내역에서 메모하기 페이지를 보고 레퍼런스로 활용해도 좋겠다고 생각했는데요, 어떻게 생각하시는지 궁금합니다.

> 토스 모임통장 메모하기
> <img src=https://github.com/rbgus2002/studygroup/assets/62997391/d13d16aa-4210-4deb-a6c3-5bb9925fb211 width=250> 

> 뜨끔 현재 화면
> <img src=https://github.com/rbgus2002/studygroup/assets/62997391/666a3a91-b45a-45ea-bb9e-e5dd1a3a740c width=250>

<br>
기록 박스 우측 하단에 기록 글자수와 같은 라인에 회색 작은 글씨로 ```스터디원 모두가 볼 수 있는 기록이에요```라고 달아놓으면 좋지 않을까 싶네요!

**Comment by Arkady1356** [comment]
오호 오호 !! 피드백 들어온거 신기하다 ㅋㅋ 어떤 루트로 들어왔는지도 궁금해요 ~

---

유사 내용이 #206 에서 이야기 되었었네요..!  

  
> 저도 비슷하게 생각하는데, 이 부분은 스터디원 모두가 볼 수 있음을 알리는 문구가 추가 되어야 한다고 생각해요.

동의합니다 !! 하지만

> 기록 박스 우측 하단에 기록 글자수와 같은 라인에 회색 작은 글씨로 ```스터디원 모두가 볼 수 있는 기록이에요```라고 달아놓으면 좋지 않을까 싶네요!

는 난잡한 느낌일 것 같아서 지양하고 싶습니다 ㅎㅎ.. 
만약 디자이너와 이야기 할 수 있다면 디자이너의 의견을 따라가는 것이 맞을 것 같고, 
아니면 입력 힌트인 '자세한 기록을 적어주세요'의 멘트를 수정해서 모두가 함께한 기록을 작성한다는 느낌으로 개선해도 좋을 것 같아요..!!

**Comment by rbgus2002** [comment]
> 오호 오호 !! 피드백 들어온거 신기하다 ㅋㅋ 어떤 루트로 들어왔는지도 궁금해요 ~

친구가 개인 공부 기록용으로 뜨끔 사용하고 있는데, 직접 카톡으로 말해줬어요 !

---

오호 어디서 논의했는지 까먹고 있었는데 🫨👍
흠 말씀해주신대로 문구를 추가하면 UI적으로 더 난잡해지는 것은 피할 수 없을 거 같습니다! 당장은 디자이너가 없으니 저희가 디자이너라고 생각하고 나아가시져 ㅎㅎ 

> 아니면 입력 힌트인 '자세한 기록을 적어주세요'의 멘트를 수정해서 모두가 함께한 기록을 작성한다는 느낌으로 개선해도 좋을 것 같아요..!!

이 생각도 해봤는데 기록에 글이 적혀져있는 경우에 확인할 수 없으니 고려사항에서 제외했는데, 그러면 난잡해지는 UI를 생각해서 입력힌트에 `스터디원 모두가 볼 수 있는 기록이에요`라고 수정하면 어떨까요?

변경 전 : `자세한 기록을 적어주세요`
변경 후 : `스터디원 모두가 볼 수 있는 기록이에요`

**Reply by Arkady1356** [reply]
<img width="234" alt="image" src="https://github.com/user-attachments/assets/5ddb06cc-0153-4c7b-a3b0-cf3248a282e7">
어떤가욥?

**Reply by rbgus2002** [reply]
저는 asis보다 훨씬 나은 UX라고 생각듭니다 ㅎㅎ

**Reply by Arkady1356** [reply]
ㅋㅋㅋ 바로 issue 발행 및  반영하겠습니다 ~
